app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
```suggestion
  def home; end
```
це коротша форма, використовуй її поки екшн пустий